src/python/pants/backend/codegen/protobuf/protoc.py

```diff
@@ -13,6 +13,10 @@ class Protoc(TemplatedExternalTool):
 
     default_version = "3.20.1"
     default_known_versions = [
+        "25.2|linux_arm64  |07683afc764e4efa3fa969d5f049fbc2bdfc6b4e7786a0b233413ac0d8753f6b|3071655",
+        "25.2|linux_x86_64 |78ab9c3288919bdaa6cfcec6127a04813cf8a0ce406afa625e48e816abee2878|3105555",
+        "25.2|macos_arm64  |8822b090c396800c96ac652040917eb3fbc5e542538861aad7c63b8457934b20|2209071",
+        "25.2|macos_x86_64 |5fe89993769616beff1ed77408d1335216379ce7010eee80284a01f9c87c8888|2235343",
```

Comment:
Just to match the older lines.
```suggestion
        "25.2|linux_arm64 |07683afc764e4efa3fa969d5f049fbc2bdfc6b4e7786a0b233413ac0d8753f6b|3071655",
        "25.2|linux_x86_64|78ab9c3288919bdaa6cfcec6127a04813cf8a0ce406afa625e48e816abee2878|3105555",
        "25.2|macos_arm64 |8822b090c396800c96ac652040917eb3fbc5e542538861aad7c63b8457934b20|2209071",
        "25.2|macos_x86_64|5fe89993769616beff1ed77408d1335216379ce7010eee80284a01f9c87c8888|2235343",
```